yarn-project/circuits.js/src/utils/serialize.ts

```diff
@@ -181,4 +181,4 @@ export function toFriendlyJSON(obj: object): string {
     },
     2,
   );
-}
\ No newline at end of file
+}
```

Comment:
Prettier should put a line at the end of files so this change seems to be caused by incorrect formatting. We should probably configure prettier so that it throws an error instead of warning so that we keep the formatting coherent across codebase.
<img width="529" alt="image" src="https://user-images.githubusercontent.com/13470840/229826626-d6bc2595-ad0b-43a6-8ffd-96ca7ca96b4b.png">

Reply:
should probably add `yarn formatting` as a `preCommit` step

Reply:
Hmm I'm running the formatter in circuits.js locally and it's not complaining, and the CI is not reporting any warnings either (see [here](https://app.circleci.com/pipelines/github/AztecProtocol/aztec3-packages/1105/workflows/dea8b15d-4c84-480c-a0a4-d70a60d1e680/jobs/8052)).

>~/Projects/aztec3-packages/yarn-project/circuits.js (palla/circuits-tests-ci)$ yarn formatting
>Checking formatting...
>All matched files use Prettier code style!